xboxone/xboxone.py

```diff
@@ -43,7 +43,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_DEVICE): cv.string,
-    vol.Optional(CONF_IP_ADDRESS, default=None): cv.string,
+    vol.Optional(CONF_IP_ADDRESS, default='0.0.0.0'): cv.string,
```

Comment:
If we default to this, we should also not send the IP option in the /devices request below. What if we just default to empty string? That is still falsey, right?

Reply:
Agreed, its fixed! Empty string is good cause the if-check for it will return False, so it does not get appended to /device call